qlib/linux_def.rs

```diff
@@ -3037,6 +3038,238 @@ impl MemoryDef {
     pub const KERNEL_END_P2_ENTRY: usize = (Self::PHY_UPPER_ADDR / Self::ONE_GB) as usize; //512
 }
 
+
+#[cfg(feature = "cc")]
+pub struct MemoryDef {}
+
+#[cfg(feature = "cc")]
+impl MemoryDef {
```

Comment:
This seems as a lot of reduplication from the non-cc case. Did you try to cluster only the parts that differ? That would help readability.

Reply:
Oh, I was looking at older commits.